omegaconf/omegaconf.py

```diff
@@ -467,6 +467,23 @@ def clear_resolvers() -> None:
         BaseContainer._resolvers = {}
         register_default_resolvers()
 
+    @classmethod
+    def clear_resolver(cls, name: str) -> bool:
+        """Clear(remove) any resolver only if it exists. Returns a bool: True if resolver is removed and False if not removed.
+
+        .. warning:
+            This method can remove deafult resolvers as well.
+
+        :param name: Name of the resolver.
+        :return: A bool (``True`` if resolver is removed, ``False`` if not found before removing).
+        """
+        if cls.has_resolver(name):
+            _ = BaseContainer._resolvers.pop(name)
```

Comment:
Why not just ignore the return value?
```suggestion
            BaseContainer._resolvers.pop(name)
```